ios/HathorMobile/Base.lproj/LaunchScreen.xib

```diff
@@ -1,37 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I guess that the splash screen is broken on this PR but @pedroferreira1 fixed it on https://github.com/HathorNetwork/hathor-wallet-mobile/pull/166